data/json/furniture_and_terrain/appliances.json

```
@@ -106,6 +106,29 @@
     ],
     "flags": [ "CIRCLE_LIGHT", "APPLIANCE", "ENABLED_DRAINS_EPOWER", "CTRL_ELECTRONIC" ]
   },
+  {
+    "id": "ap_standing_floodlight",
+    "type": "vehicle_part",
+    "name": { "str": "standing floodlight" },
+    "item": "standing_floodlight",
+    "location": "structure",
+    "symbol": "*",
+    "broken_symbol": "-",
+    "categories": [ "lighting" ],
+    "color": "white",
+    "broken_color": "blue",
+    "durability": 20,
+    "description": "A very bright, circular light that illuminates the area outside the vehicle when turned on.",
```

```suggestion
    "description": "A very bright, circular light that illuminates the area around it when switched on.",
```
This should probably use `copy-from` to the floodlight vehicle part since this is basically a vehicle floodlight on a stick

---

Great thanks! This does make it use the "Floodlight" vehicle tile asset mind which may or may not be desirable